less/mixins.less

```diff
@@ -1 +1,13 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+
+.button-format (@w, @x, @y, @z) {
+    width: 100%;
+    background-color: @w;
+    color: @x;
```

Comment:
Good job creating a parametric mixin, but anyone working on your code or trying to use it wont be very happy by your naming convention. You need to use more specific detailed and applicable names for these parameters, like `bgc`, `textcolor` `fontsize` and `borderrad`.